requirements_all.txt

```diff
@@ -373,6 +373,9 @@ colorlog==4.0.2
 # homeassistant.components.concord232
 concord232==0.15
 
+# homeassistant.components.coned
+coned==0.1.11
```

Comment:
Doesn't match the [current module version](https://github.com/bvlaicu/coned/blob/master/coned/__init__.py#L9).

Reply:
Done. Version was wrong on the pypi module side. Thanks.